pkg/cmd/pulumi/stack_init_test.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValidateCreateStackOpts(t *testing.T) {
+	t.Parallel()
+
+	var cases = []struct {
+		name                 string
+		rawTeams, validTeams []string
+		hasValidTeams        bool
+	}{
+		{
+			name: "Input Is Empty",
+			// no raw or valid teams
+			rawTeams:   []string{},
+			validTeams: []string{},
+		},
+		{
+			name:       "a aingle valid team is provided",
+			rawTeams:   []string{"TeamRocket"},
+			validTeams: []string{"TeamRocket"},
+		},
+		{
+			name:       "only invalid teams are provided",
+			rawTeams:   []string{" ", "\t", "\n"},
+			validTeams: []string{},
+		},
+		{
+			name:       "mixed valid and invalid teams are provided",
+			rawTeams:   []string{" ", "Edward", "\t", "Jacob", "\n"},
+			validTeams: []string{"Edward", "Jacob"},
```

Comment:
Do we have a spec from the service as to what's actually a valid team name? Is "Robert'); DROP TABLE Students;--" a valid team name?

Reply:
Lets just let service do the validation. There's no need for us to validate them, just do the simple trim of whitespace and empty.

Reply:
That was the intended goal of this test. I just wanted to validate if users sent values with whitespace or empty, we would optimistically strip whitespace.

Reply:
Would you like me to add a comment clarifying that intent?

Reply:
"Why" comments never hurt

Reply:
I expanded the existing comment to explain the intent of the test and the motivation. Resolving this conversation as a result.